src/main/java/org/passay/AllowedCharacterRule.java

```diff
@@ -11,14 +11,14 @@
  *
  * @author  Middleware Services
  */
-public class AllowedCharacterRule implements Rule
+public class AllowedCharacterRule implements CharacterData, Rule
```

Comment:
The problem solves is the ability to get a CharSequence out of similar rules such as AllowedCharacterRule and CharacterRule, otherwise I needed to use getAllowedCharacters and getValidCharacters which in essence does the same thing.  I referred to this under the refactoring message.